README.md

```diff
@@ -204,3 +204,13 @@ godep save ./...
 git add Godeps vendor
 git commit
 ```
+
+Adding RPCs
+-----------
+
+Boulder is moving towards using gRPC for all RPCs. To add a new RPC method, add
+it to the relevant .proto file, then run:
+
+    docker-compose run boulder go generate ./path/to/pkg/...
```

Comment:
Is this better in `CONTRIBUTING.md`? I think it would also be a good chance to describe why its important to generate from within the docker container to avoid having to match your protobuf compiler version with the one used by CI.